packages/apps-config/src/api/rpc/moonbeam.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { DefinitionRpc } from '@polkadot/types/types';
+
+export default function ():Record<string, Record<string, DefinitionRpc>> {
+  const dummyDescription = {
```

Comment:
If these contain nothing, i.e. is not usable from the RPC page, we should leave it out completely. The console warning is harmless and doesn't affect operation. However, with inaccurate information here, it means it is available from the RPC page and calls won't work, so it is the worse option.

Reply:
@jacogr I added the real rpc declarations. let me know what you think

Reply:
All good.